s4/more_str_meth.rb

```diff
@@ -0,0 +1,29 @@
+# More methods to concatenate.
+a = "A"; b = "B"; c = ""
+
+# Usual
+p a+b
+
+# Arithmetically
+c +=(a+b); p c;
+
+# Con .concat(), << y .prepend() se asigna
+# en la variable a la concatenación.
+
+# .concat()
+c = a.concat(b); p c
+a = "A"
+
+# Shovel operator (<<)
+p a << b
+a = "A"
+
+# prepend()
+p a.prepend(b)
```

Comment:
`prepend `concatenates at the start, `concat `adds the string at the end as usual, and the Shovel operator, in this case, doesn’t make much difference, but if it were arrays, it would add the entire array as one more object from the other array.